lib/asyn-thread.c

```diff
@@ -667,7 +667,7 @@ int Curl_resolver_getsock(struct connectdata *conn,
 #endif
     ms = Curl_timediff(Curl_now(), reslv->start);
     if(ms < 3)
-      milli = 0;
+      milli = ms;
```

Comment:
why this?

Reply:
Sorry, we have a better fix. I remove this patch from pull request, pleaase ignore it.